### ISSUE TITLE
lib folder confusion

### ISSUE THREAD

**Issue by melanieseltzer** [flexdinesh/npm-module-boilerplate#1]
Hi! Quick question, just trying to understand. In your build script the output is set to `lib` for when Babel transpiles. And the entry point for the package is `./lib/index.js` under the main in package.json. Makes sense. But then in .npmignore the `lib` directory is getting ignored... but wouldn't you need npm to not ignore the `lib` directory when publishing the package? To get the transpiled code on install of the package?

Many thanks 😄

**Comment by flexdinesh** [comment]
@melanieseltzer `.npmignore` is not supposed to ignore `lib` dir. When I created the `.npmignore` file I copied everything from `.gitignore` and I should have removed `lib` but I didn't, apparently. But what's surprising is my other npm lib [Typy](https://github.com/flexdinesh/typy) has the same config and everything seems to work fine. I guess npm is intelligent enough to figure out `lib` dir should not be ignored because of the `"main": "./lib/index.js"` entry in `package.json`. 

I'm going to fix this and push a new commit. Thanks for bringing it up. 🎉
